zebra-consensus/src/transaction.rs

```diff
@@ -457,31 +457,31 @@ where
         let mut spent_utxos = HashMap::new();
         let mut spent_outputs = Vec::new();
         for input in inputs {
-            match input {
-                transparent::Input::PrevOut {
-                    outpoint,
-                    unlock_script: _,
-                    sequence: _,
-                } => {
-                    tracing::trace!("awaiting outpoint lookup");
-                    let utxo = if let Some(output) = known_utxos.get(outpoint) {
-                        tracing::trace!("UXTO in known_utxos, discarding query");
-                        output.utxo.clone()
+            if let transparent::Input::PrevOut {
+                outpoint,
+                unlock_script: _,
+                sequence: _,
+            } = input
+            {
```

Comment:
Optional: removing the two unused fields makes this a bit more concise, but it's subjective whether it's better or not :sweat_smile: 

```suggestion
            if let transparent::Input::PrevOut { outpoint, .. } = input {
```

Reply:
I think shorter is more readable, but I don't mind either way.